script.js

```diff
@@ -1,12 +1,68 @@
 //You can edit ALL of the code here
 function setup() {
-  const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
+const rootElem = document.getElementById("root");
+const allEpisodes = getAllEpisodes();
 
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  rootElem.innerHTML = "";
+  const display = document.getElementById("display");
+  display.innerText = `Display ${episodeList.length}/${allEpisodes.length}`;
+  episodeList.forEach(e => {
+  const boxBody = document.createElement('div');
+  boxBody.id = "boxBody";
+  const title = document.createElement('h3');
+  title.id = "title";
+  const imgBox = document.createElement('img');
+  imgBox.id = "imgBox";
+  const para = document.createElement('p');
+  para.id = "para"
+  rootElem.appendChild(boxBody);
```

Comment:
Generally it's better (for performance reasons) to append all your sub-elements first, and then append to the DOM (in your case the root) only once.